test/test_bandedlu.jl

```diff
@@ -24,11 +24,7 @@ let
     @test BandedMatrices._promote_to_blas_type(typ, Int16)      == typ
     @test BandedMatrices._promote_to_blas_type(typ, Int8)       == typ
 
-    if VERSION < v"0.5.0-dev+5122"
```

Comment:
@dlfivefifty if I have to bisect on a bug in base that involves this package, this will save time and hassle of getting unrelated failures and avoid needing to simultaneously track versions of multiple repos at the same time. so please leave this in.